store/src/java/com/zimbra/cs/mailbox/MailSender.java

```diff
@@ -1202,11 +1206,13 @@ private String getNextHost() {
     }
 
     private void sendMessageToHost(String hostname, MimeMessage mm, Address[] rcptAddresses)
-    throws MessagingException {
-        mSession.getProperties().setProperty("mail.smtp.host", hostname);
+    throws MessagingException, ServiceException {
+    	    mSession.getProperties().setProperty("mail.smtp.host", hostname);
```

Comment:
nit: shouldn't have tabs in the source code, but looks like one snuck in here

Reply:
Fixed eclipse config and applied the format.